src/account/azext_account/tests/latest/recordings/test_account.yaml

```diff
@@ -12,18 +13,18 @@ interactions:
       Connection:
       - keep-alive
       Content-Length:
-      - '174'
+      - '199'
       Content-Type:
       - application/json
       ParameterSetName:
-      - --name --billing-scope --display-name --workload
+      - --name --billing-scope --display-name --workload --reseller-id
       User-Agent:
-      - AZURECLI/2.11.1 azsdk-python-subscriptionclient/unknown Python/3.6.8 (Windows-10-10.0.19041-SP0)
+      - AZURECLI/2.13.0 azsdk-python-subscriptionclient/unknown Python/3.6.8 (Windows-10-10.0.19041-SP0)
     method: PUT
     uri: https://management.azure.com/providers/Microsoft.Subscription/aliases/cli_alias000001?api-version=2020-09-01
   response:
     body:
-      string: '{"id":"/providers/Microsoft.Subscription/aliases/cli_alias000001","name":"cli_alias000001","type":"Microsoft.Subscription/aliases","properties":{"subscriptionId":"00000000-0000-0000-0000-000000000000","provisioningState":"Accepted"}}'
+      string: '{"id":"/providers/Microsoft.Subscription/aliases/cli_alias000001","name":"cli_alias000001","type":"Microsoft.Subscription/aliases","properties":{"subscriptionId":"89b84889-2fa2-40f4-a7c1-f2f0de19ac39","provisioningState":"Accepted"}}'
```

Comment:
why this subs id not masked?

Reply:
subscription id is only masked in limited cases by [azure_devtools](https://github.com/Azure/azure-python-devtools/blob/4be812e0237572afa23c37c259c414be79557e54/src/azure_devtools/scenario_tests/recording_processors.py#L50-L69). I may have manually masked it last time.